trade-finance-manager-ui/component-tests/utilisation-reports/utilisation-report-reconciliation-for-report.component-test.ts

```diff
@@ -61,210 +63,200 @@ describe(page, () => {
-    wrapper.expectElement(markAsDoneButtonSelector).doesNotHaveClass('govuk-button--secondary');
+      const markAsDoneButtonSelector = `${keyingSheetTabSelector} [data-cy="keying-sheet-mark-as-done-button"]`;
+      wrapper.expectInput(markAsDoneButtonSelector).toHaveValue('Mark as done');
+      wrapper.expectElement(markAsDoneButtonSelector).toHaveAttribute('formaction', `/utilisation-reports/${reportId}/keying-data/mark-as-done`);
+      wrapper.expectElement(markAsDoneButtonSelector).doesNotHaveClass('govuk-button--secondary');
 
-    const markAsToDoButtonSelector = `${keyingSheetTabSelector} [data-cy="keying-sheet-mark-as-to-do-button"]`;
-    wrapper.expectInput(markAsToDoButtonSelector).toHaveValue('Mark as to do');
-    wrapper.expectElement(markAsToDoButtonSelector).toHaveAttribute('formaction', `/utilisation-reports/${reportId}/keying-data/mark-as-to-do`);
-    wrapper.expectElement(markAsToDoButtonSelector).hasClass('govuk-button--secondary');
+      const markAsToDoButtonSelector = `${keyingSheetTabSelector} [data-cy="keying-sheet-mark-as-to-do-button"]`;
+      wrapper.expectInput(markAsToDoButtonSelector).toHaveValue('Mark as to do');
+      wrapper.expectElement(markAsToDoButtonSelector).toHaveAttribute('formaction', `/utilisation-reports/${reportId}/keying-data/mark-as-to-do`);
+      wrapper.expectElement(markAsToDoButtonSelector).hasClass('govuk-button--secondary');
 
-    wrapper.expectElement(`${keyingSheetTabSelector} table[data-cy="keying-sheet-table"]`).toExist();
-  });
+      wrapper.expectElement(`${keyingSheetTabSelector} table[data-cy="keying-sheet-table"]`).toExist();
+    });
 
-  it('should render the keying sheet tab with headings, text, table but not buttons for PDC_READ users', () => {
-    const wrapper = getWrapper({ ...params, user: aPdcReadUser() });
-    const keyingSheetTabSelector = 'div#keying-sheet';
+    it('should render the keying sheet tab with headings, text, table but not buttons for PDC_READ users', () => {
+      const wrapper = getWrapper({ ...params, user: aPdcReadUser() });
+      const keyingSheetTabSelector = 'div#keying-sheet';
 
-    wrapper.expectElement(keyingSheetTabSelector).toExist();
+      wrapper.expectElement(keyingSheetTabSelector).toExist();
 
-    wrapper.expectText(`${keyingSheetTabSelector} h2[data-cy="keying-sheet-heading"]`).toRead('Keying sheet');
-    wrapper.expectText('[data-cy="select-payments-text"]').notToExist();
-    wrapper
-      .expectText('[data-cy="payments-on-premium-payments-tab-text"]')
-      .toMatch(/Payments on the premium payments tab will show as reconciled when they have been marked as done here./);
+      wrapper.expectText(`${keyingSheetTabSelector} h2[data-cy="keying-sheet-heading"]`).toRead('Keying sheet');
+      wrapper.expectText('[data-cy="select-payments-text"]').notToExist();
+      wrapper
+        .expectText(`${keyingSheetTabSelector} p`)
```

Comment:
I think this selector has changed incorrectly in the merge

Reply:
Yep, spotted this too - pushed a change at the same time